tests/ginkgo/sequential/1-051_validate_argocd_agent_principal_test.go

```diff
@@ -51,8 +53,8 @@ import (
 var _ = Describe("GitOps Operator Sequential E2E Tests", func() {
 
 	const (
-		argoCDName               = "argocd"
-		argoCDAgentPrincipalName = "argocd-agent-principal"
+		argoCDName               = "openshift-gitops"
```

Comment:
The `openshift-gitops` `ArgoCD` CR name is an invention of the downstream `gitops-operator` repository. Notice that there are no other references to `openshift-gitops` within this codebase. 

We should thus not use it here, as the argocd-operator repo is upstream of `gitops-operator` (and argocd-operator is vendor neutral, whereas openshift-gitops operator is not).

Reply:
My bad :) I used a different name now, PTAL